### PR TITLE
Add interview with Iva F

### PR DESCRIPTION
**Pull request by anezkamll**

### CODE REVIEW

File: content/blog/iva-fingerova-interview.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+I’ve always been the underdog - since the first day of elementary school perhaps even earlier. I guess that's why I have such an urge to get the underdogs to thrive, make them see that they are more than enough as they are, and show them that bullshitting never gets long-term results. 
+
+Thanks to all these opportunities I could define myself as a person that can work mainly with the growth mindset people in a transparent and maximum inclusive environment that supports individuals. And I’m so happy I found such a place!
+
+**Your current role has an impressive title - Head of Product Development at Invity.io, a company developing cryptocurrency rate comparison tool, where you’re currently building a new team. How’s that going? What’s most challenging in your current role? And would you mind telling us about the project you’re working on?**
```
Comment (dariagrudzien) [inline content/blog/iva-fingerova-interview.md:45]: I would add a link to https://invity.io/ here

File: content/blog/iva-fingerova-interview.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Well, Invity was just founded when I joined the team. At that time I was a bit shaken after my previous job and I was so glad to step down from leadership and slowly gather my strength. During those 15 months or so I channeled almost all my previous gigs. It was a rejuvenating process - I was a client, a PM, UX designer and UI designer (ehm), tester, researcher, and more, to make sure things were done in a “good enough” way. It was all necessary and I got to appreciate everything I’ve learned in my past and it helped to make invity.io a success.
```
Comment (dariagrudzien) [inline content/blog/iva-fingerova-interview.md:47]: At the end of this paragraph `invity.io` is written with small letter. I think it should be capitalized?

**Review by dariagrudzien — APPROVED** [review]
In general everything looks good, added two tiny nitpicks. Once you have the image ready, this will be good to go. 

Not sure if you've used this feature yet - you can preview the page before merging & deploying. Look for the `Details` button of the last check on the PR. Let me know if you have any questions.

🚀

**Comment by anezkamll** [comment]
Thank you for the review, unfortunately I won't be able to create the header - my laptop died in an unfortunate cat-caused accident Sunday early morning so I don't have any proper HW to work on (temporarily, I hope). Could you please help me again this time and create it? I can send you Iva's photo

> Not sure if you've used this feature yet - you can preview the page before merging & deploying. Look for the `Details` button of the last check on the PR. Let me know if you have any questions.
> 
Yes I used it even last time with Lenka S when you first showed me this feature :+1:  Highly appreciated. And actually thanks to this feature I realized that there is something wrong with the header image, that there is just the image, no additional graphics :)  
No questions here.

**Comment by dariagrudzien** [comment]
> unfortunate cat-caused accident

I had one of those. Sorry to hear about the cat-caused distraction!

I took the easy way out and pushed the commit into your branch. Feel free to merge if you like it.

**Comment by anezkamll** [comment]
> I took the easy way out and pushed the commit into your branch. Feel free to merge if you like it.

I don't see any photo in the preview but I think I know where the problem is (the name of the image I guess), I will fix it asap

**Comment by anezkamll** [comment]
Looking good, thank you for all the help!
Merging!

**Pull request merged**